engine/src/flutter/impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -78,20 +78,21 @@ static size_t GetVKClearValues(
 SharedHandleVK<vk::RenderPass> RenderPassVK::CreateVKRenderPass(
     const ContextVK& context,
     const SharedHandleVK<vk::RenderPass>& recycled_renderpass,
-    const std::shared_ptr<CommandBufferVK>& command_buffer) const {
+    const std::shared_ptr<CommandBufferVK>& command_buffer,
+    bool is_swapchain) const {
```

Comment:
Instead of passing in a bool can you just have `CreateVKRenderPass()` and `CreateSwapchainVKRenderPass()`?